_rules/SC1-3-1-aria-hidden-focus.md

```diff
@@ -42,7 +43,7 @@ By adding `aria-hidden="true"` to an element, content authors ensure that assist
 
 A focusable element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making it's state of visible or hidden unclear.
 
-- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
```

Comment:
If both SCs are concerned, put the 4.1.2 link back in the background section :-)